tests/test_meteodatalab/conftest.py

```diff
@@ -30,9 +30,12 @@ def machine():
 @pytest.fixture
 def data_dir(request, machine):
     """Base data dir."""
-    if machine != "tsa":
+    if machine == "tsa":
+        base_dir = Path("/project/s83c/rz+/icon_data_processing_incubator/")
+    elif machine == "balfrin":
+        base_dir = Path("/scratch/mch/ckanesan/")
```

Comment:
Yes, it could but I mentioned during the meeting that I'm trying to figure out a sensible way to reduce the amount of data before moving it. However, I can see that there's a full COSMO-1E run in that folder already. Is there a documented policy regarding storage at CSCS?

Reply:
https://confluence.cscs.ch/pages/viewpage.action?pageId=785285325
This was published for store but I don't know if it's also valid for store_new. If it is, the dataset is problematic due to the 50k files per TB rule. I might need to tar it.